python/lsst/pipe/tasks/makeCoaddTempExp.py

```diff
@@ -319,7 +319,7 @@ def runDataRef(self, patchRef, selectDataList=[]):
             if any(exps.values()):
                 dataRefList.append(tempExpRef)
             else:
-                self.log.warn("Warp %s could not be created", tempExpRef.dataId)
+                self.log.error("Warp %s could not be created", tempExpRef.dataId)
```

Comment:
FWIW, this warning has been seen pretty frequently in the HSC-RC2 reprocessing. One case is despite the ccd area overlaps with the sky patch, there are no good pixels in that patch (e.g. pixels masked out as bad).   In a way it's not really an "error" because we could expect it and we chose not to write out a warp file in that case.  In fact I'd argue for writing out a (~empty) file; that would make the execution workflow easier in the future middleware and production system.

Reply:
An old example with illustration plots can be found [on this page](https://confluence.lsstcorp.org/display/~hchiang2/Collection+of+edge+cases)   (sorry -- rotate the images by 180 deg in your brain and that's the lower-left vertical ccd in the sky)

Reply:
Thanks for the details. I'll make this back into a `warn`, but this suggests that we should do a better job managing this output, so that we either don't warn (if there's a good reason the warp wasn't made) or error/fail (if the warp should have been made but wasn't).

I've made a ticket for this: https://jira.lsstcorp.org/browse/DM-16591

Reply:
Thanks for filing the ticket!